Ted Talk/Ted Talk/ViewController/DetailTedTalkViewController.swift

```diff
@@ -9,31 +9,39 @@ import UIKit
 import WebKit
 
 class DetailTedTalkViewController: UIViewController {
-
+    
     @IBOutlet weak var titleDetail: UILabel!
     @IBOutlet weak var ofviewDetail: UILabel!
     @IBOutlet weak var nameDetail: UILabel!
     @IBOutlet weak var descriptionDetail: UILabel!
     @IBOutlet weak var tagDetail: UILabel!
     @IBOutlet weak var tedTalkWeb: WKWebView!
     
-    var tedTalk: TedTalk?
+    var tedTalk: TedTalk? = nil
     var tagString: String = "Tags: "
     
+    func setTedTalk(talk: TedTalk){
+        self.tedTalk = talk
+    }
     override func viewDidLoad() {
         
-        titleDetail.text = tedTalk?.title
-        tedTalkWeb.load(URLRequest(url: URL(string: tedTalk!.url)!))
-        tagString = tagString + (tedTalk?.tags[0])!
+        super.viewDidLoad()
+        
+        guard let displayTedTalk = tedTalk else {
```

Comment:
Even this works in order to avoid using force unwrapping, in this scenario I would recommend the following either two options:
1) Instead of using a guard, use if let and do all the configuration inside the if clause OR
2) Move all this logic (between lines 30 and 45) into another private and auxiliar method, like: setupTedTalk